bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -155,6 +160,8 @@ void DataAggregator::findPerfExecutable() {
 }
 
 void DataAggregator::start() {
+  std::string ItracePerfScriptArgs;
```

Comment:
Is there a reason to declare `ItracePerfScriptArgs` in this scope?

Reply:
I just saw other std::string being declared at the beginning of functions. I've moved the declaration into the if statement scope.